src/main.rs

```diff
@@ -58,24 +71,45 @@ async fn main() -> Result<(), Termination> {
 
     let opts = CliOpts::from_args();
     let CliOpts {
+        dst_addr,
+        identity_addr,
         endpoints,
         overrides,
-        addr,
         endpoints_dir,
+        identities_dir,
     } = opts;
-    tracing::debug!(?endpoints, ?overrides, ?addr, ?endpoints_dir);
+    tracing::debug!(
+        ?dst_addr,
+        ?identity_addr,
+        ?endpoints,
+        ?overrides,
+        ?endpoints_dir,
+        ?identities_dir
+    );
+
+    match identities_dir {
+        Some(identities) => {
+            let svc = IdentityService::new(identities)?;
+            tokio::spawn(svc.serve(identity_addr));
+        }
+        None => {
+            let svc = IdentityService::empty();
+            tokio::spawn(svc.serve(identity_addr));
+        }
+    }
```

Comment:
nit/tioli: seems like this could just be
```suggestion
    let identity_svc = match identities_dir {
        Some(identities) => IdentityService::new(identities)?,
        None => IdentityService::empty(),
    };
    tokio::spawn(identity_svc.serve(identity_addr));
```